### DISCUSSION TITLE
Plantuml not working with Epsilon Eclipse Distribution

### DISCUSSION THREAD

**Discussion by jgsuess** [hallvard/plantuml#182] [Q&A]
When I install the plugin into the latest version of Epsilon which I have installed using the installer: https://eclipse.dev/epsilon/download/ the plugin fails and my providers are broken:
![image](https://github.com/user-attachments/assets/aaacfb9a-a95a-4ab8-8476-3b2b2d8f7713)

The settings seem fine:
![image](https://github.com/user-attachments/assets/f02677b4-913c-4aae-9338-b416eccd7898)

I saw a requirement in the log saying that GEF-DOT was required. I installed that hoping it would address the issue, but did not make a difference.

I am a bit stuck, as I am using Plantuml to do documentation of Ecore models.

Any advice?

**Comment by travkin79** [comment]
HI @jgsuess,
Are you seeing any entries in the error log view? Is there a stacktrace?
Graphviz / DOT seems to work as expected.
Does it work with an older Eclipse (it was implemented / tested with Eclipse 2023-12)?
